packages/gestalt/src/Video.js

```diff
@@ -28,6 +28,7 @@ type Props = {|
   accessibilityMuteLabel: string,
   accessibilityPauseLabel: string,
   accessibilityPlayLabel: string,
+  accessibilityProgressbarLabel: string,
```

Comment:
nit: should this use "ProgressBar" since those are typically two separate words?

Reply:
Definitely - updates